.github/workflows/docs.yml

```diff
@@ -23,28 +16,22 @@ concurrency:
 jobs:
   # Single deploy job since we're just deploying
   deploy:
-    environment:
-      name: github-pages
-      url: ${{ steps.deployment.outputs.page_url }}
     runs-on: ubuntu-latest
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - name: Fetch gh-pages branch
+        run: git fetch origin gh-pages
```

Comment:
This step is not necessary, can be done in the previous step. [See here](https://github.com/actions/checkout#checkout-a-different-branch).